src/index.html

```diff
@@ -3,11 +3,667 @@
+        <div class="container">
+          <h2 class="our-expertise__title">
+            Our expertise
+          </h2>
+
+          <div class="our-expertise__content">
+            <div class="our-expertise__card">
+              <img
+                src="./images/our-expertise/branding.jpg"
+                alt="Branding"
+                class="our-expertise__card-img"
+              >
+
+              <h3 class="our-expertise__card-title">
+                Branding
+              </h3>
+
+              <p class="our-expertise__card-desc">
+                We create additional value for companies, products, services as well as verbal and visual ways to deliver it to the audience.
+              </p>
+            </div>
+
+            <div class="our-expertise__card">
+              <img
+                src="./images/our-expertise/communication.jpg"
+                alt="Communication"
+                class="our-expertise__card-img"
+              >
+
+              <h3 class="our-expertise__card-title">
+                Communication
+              </h3>
+              <p class="our-expertise__card-desc">
+                We strive to create communications that can increase media performance. We use everything — words, meanings, stories, art, movies.
+              </p>
+            </div>
+
+            <div class="our-expertise__card">
+              <img
+                src="./images/our-expertise/strategy.jpg"
+                alt="Strategy"
+                class="our-expertise__card-img"
+              >
+
+              <h3 class="our-expertise__card-title">
+                Strategy
+              </h3>
+
+              <p class="our-expertise__card-desc">
+                We create business growth strategies, from the moment of its birth to the achievement of the necessary business indicators.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section process" id="services">
+        <div class="container">
+          <div class="grid grid--block">
+            <div class="
+              grid__item
+              grid__item--tablet-1-3
+              grid__item--small-desktop-1-4
+              grid__item--desktop-1-4
+              grid__item--large-screen-1-4
+            ">
+              <div class="process__info">
+                <h3 class="process__title-small">
+                  Services
+                </h3>
+
+                <h2 class="process__title">
+                  Air is a full service creative agency
+                </h2>
+
+                <p class="process__description">
+                  Deep analytics, strong strategy and bright creative ideas.
+                  <br>
+                  <br>
+                  We are sure that first-rate job is possible only if all three components are united.
+                </p>
+              </div>
+            </div>
+
+            <div class="
```

Comment:
Something like this is preferred
![image](https://github.com/mate-academy/layout_dia/assets/87986955/7d0ec0ae-d45b-4de9-a4b7-867879a5a199)
Format all similar indents